rust/src/common.rs

```diff
@@ -169,3 +179,9 @@ pub unsafe extern "C" fn rs_to_hex_sep(
     // overwrites last separator with final null char
     oslice[3 * islice.len() - 1] = 0;
 }
+
+// Defined in util-debug.c
+/// cbindgen:ignore
+extern {
+    pub fn CallbackFatalErrorOnInit(arg: *const c_char);
```

Comment:
@jasonish should we create a 'public' `SCFatalErrorOnInit` instead?

Also, would like it to behave like `FatalError` and `SCLog...` to take a format string.

Reply:
Probably, should be a macro in Rust like the other log ones.

Much easier to do if FatalErrorOnOnit can become a C function though, rather than a macro, otherwise it'll need a C wrapper function first.

Reply:
> Much easier to do if FatalErrorOnOnit can become a C function though

Failed to make a variadic fn out of this macro on the C side. Wrapper way does work as done in this PR.